packages/firebase_dynamic_links/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.5.0
+
+* **Breaking change**. Changed architecture and method names to be able to differentiate between
```

Comment:
I would include what methods are no longer available and what methods replaced them.

Reply:
Done!